### ISSUE TITLE
Show only most recent video from a subscription

### ISSUE THREAD

**Issue by phillvancejr** [FreeTubeApp/FreeTube#546]
Invidious settings have an option showing only the most recent video from a subscription.  Showing only the most recent video is nice because then on your subscription page you have one thumbnail for each subscription and with hide watched videos turned on, as you watch videos the subscription page will show the most recent unwatched video from each subscription.

**Comment by PrestonN** [comment]
You can hide subscription videos when watched by enabling it in your settings.

**Comment by phillvancejr** [comment]
@PrestonN I know that but is there a way to only show the most recent video uploaded by a subscription?  Currently on my subscription page multiple videos from subscriptions appear, however the feature I am requesting (which Invidious sites have) would limit the videos from a subscription show on the subscription page to only the most recent one. This is a separate feature from hiding watched videos, I simply mentioned it in the original post because they work well together.

**Comment by PrestonN** [comment]
Ahh, my mistake.  Yeah this is a valid request.  This can be done at some point.

**Comment by WandeingLagomorph793** [comment]
Agreed on this one, Being able to show only the most recent video would help a lot if you have many active subscriptions.
